### PR TITLE
Silver Greedy + Prefix Sum section Editorials

### PR DESCRIPTION
**Pull request by tachithanhdanh**

### CODE REVIEW

**Comment by tachithanhdanh** [comment]
I added the C++ implementation for this problem https://usaco.guide/silver/prefix-sums#problem-https://cses.fi/problemset/task/1650

**Comment by tachithanhdanh** [comment]
Could someone please review these solutions?
I think they are detailed enough

**Comment by bqi343** [comment]
What about step 3 of https://usaco.guide/general/contributing?lang=cpp#adding-a-solution ?

**Comment by tachithanhdanh** [comment]
Do I have to remove the spoiler in that file too?

**Comment by bqi343** [comment]
Remove what spoiler?

**Comment by tachithanhdanh** [comment]
Oops I misread the file

**Comment by tachithanhdanh** [comment]
Do I need to add the tags?

**Comment by bqi343** [comment]
If you want.

File: solutions/CSES Stick Lengths.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+id: cses-1074
+source: CSES
+title: Stick Lengths
+author: Danh Ta Chi Thanh
+---
+
+In this problem, we are given an array with n elements. We want to find the minimum cost to make all of the elements equal.
```
Comment (bqi343) [inline solutions/CSES Stick Lengths.mdx:8]: $n$ instead of n

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: solutions/CSES Stick Lengths.mdx (hunk 1 above)
Comment (tachithanhdanh) [reply solutions/CSES Stick Lengths.mdx]: I have changed that, thanks

**Pull request merged**